### DISCUSSION TITLE
Alertmanager not send any alerts to webhook

### DISCUSSION THREAD

**Discussion by Codecaver** [prometheus-operator/prometheus-operator#5989] [Q&A]
I use alertmanagerconfig to configure alert manager.yaml. But it doesn't take effect. I wonder if the default configuration file suppresses the alarm. Should I delete the default configuration first? The default configuration file is secret: alertmanager-prometheus-stack-univision-alertmanager

**Comment by LeoQuote** [comment]
Check your config in alertmanager config page to see there’s anything wrong. Provide the final config and your k8s resources if possible.
